conda-env/prod.yml

```diff
@@ -0,0 +1,37 @@
+# Conda production environment for datasm operations.
+name: datasm_prod
+channels:
+  - conda-forge
+  - e3sm
+  - defaults
+dependencies:
+  # Base
+  # ==================
+  - python >=3.9
+  - pip
+  - distributed
+  - ipdb
+  - matplotlib
+  - netcdf4
+  - numpy >=1.23.0 # This version of numpy includes support for Python 3.11.
+  - pyyaml
+  - termcolor
+  - tqdm
+  - watchdog
+  - xarray >=2022.02.0 # This version of Xarray drops support for Python 3.8.
```

Comment:
**Packages are considered direct dependencies if they are directly imported by `datasm`.**
Update this section by removing or adding any dependencies as needed.